.snippets/code/products/settlement/get-started/snippet-1.ts

```diff
@@ -27,13 +29,15 @@ export async function getSigner<N extends Network, C extends Chain>(
   let signer: Signer;
   const platform = chain.platform.utils()._platform;
   switch (platform) {
-    case 'Solana':
+    case "Solana":
       signer = await getSolanaSigner(
         await chain.getRpc(),
-        getEnv('MAINNET_SOL_PRIVATE_KEY')
+        getEnv("MAINNET_SOL_PRIVATE_KEY")
```

Comment:
```suggestion
       "MAINNET_SOL_PRIVATE_KEY"
```
We should remove this if there is no `getEnv` function?